src/Day.jsx

```diff
@@ -7,6 +7,14 @@ let Day = React.createClass({
 
   propTypes: TimeGrid.propTypes,
 
+  getDefaultProps() {
+    return {
+      now: new Date(),
+      type: 'gutter',
+      timeslots: 2
+    }
```

Comment:
This is the problem with re-using attributes from other components.  These properties are absolutely required for TimeGrid to function.  Thus, when clicking the buttons week/day in the toolbar, there are a slew of warnings that these proptypes are not present, and in fact the calendar fails to display the current time unless this is present.

2 choices (yours to make):

1. also use TimeGrid's default props
      getDefaultProps() {
        return TimeGrid.defaultProps
      }

2. declare these properties on the Calendar component with the proper defaults